lib/seccomp-tools/const.rb

```diff
@@ -59,6 +59,8 @@ module BPF
         KILL: 0x00000000, # alias of KILL_THREAD
         TRAP: 0x00030000,
         ERRNO: 0x00050000,
+        USER_NOTIFY: 0x7fc00000,
```

Comment:
`USER_NOTIF`, to be consistent with Linux kernel's naming.

Reply:
Ah of course